internal/configs/version2/stream.go

```diff
@@ -35,13 +35,17 @@ type StreamServer struct {
 	ProxyNextUpstream        bool
 	ProxyNextUpstreamTimeout string
 	ProxyNextUpstreamTries   int
-	HealthCheck              bool
-	HCInterval               string
-	HCPort                   int
-	HCPasses                 int
-	HCJitter                 string
-	HCFails                  int
-	HCTimeout                string
+	HealthCheck              *StreamHealthCheck
+}
+
+type StreamHealthCheck struct {
```

Comment:
// StreamHealthCheck defines a health check for a StreamUpstream in a StreamServer.